server/api/sessions.js

```diff
@@ -84,6 +113,35 @@ internals.applyRoutes = function (server, next) {
 
     server.route({
         method: 'DELETE',
+        path: '/sessions/my/{id}',
+        config: {
+            auth: {
+                strategy: 'simple',
+                scope: 'account'
```

Comment:
This shouldn't be limited to a scope. Admin users should be able to delete their sessions too.